src/EditorFeatures/CSharpTest/CodeActions/ReplacePropertyWithMethods/ReplacePropertyWithMethodsTests.cs

```diff
@@ -294,6 +294,23 @@ private int GetProp()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsReplacePropertyWithMethods)]
+        public async Task TestComputedPropWithTrailingTriviaAfterArrow()
+        {
+            await TestAsync(
+@"class C
+{
+    public int [||]Prop => /* return 42 */ 42;
+}",
+@"class C
+{
+    public int GetProp()
+    {
+        return /* return 42 */ 42;
+    }
+}", compareTokens: false);
+        }
+
```

Comment:
outside the scope of this fix, but can we maintain expression bodied members as such during this transform?